frontend/src/App.js

```diff
@@ -44,6 +44,13 @@ GROUP BY committer_email, month, repo_id`,
   }
 
   render() {
+    let ok;
```

Comment:
wow. why not
```js
const ok = this.state.response && this.state.response.status === 200
```
?

try-catch isn't very javascript way to do such things.

or even better:
```js
const { response } = this.state;
const ok = response && response.status === 200;
```